apcd-cms/src/apps/admin_regis_table/templates/list_registrations.html

```diff
@@ -3,8 +3,8 @@
 
 {% block content %}
 
-<link rel="stylesheet" href="{% static 'admin_regis_table/registrations_table.css' %}" type="text/css">
-<link rel="stylesheet" href="{% static 'admin_regis_table/registrations_admin_modal.css' %}" type="text/css">
+<link rel="stylesheet" href="{% static 'admin_regis_table/css/table.css' %}">
+<link rel="stylesheet" href="{% static 'admin_regis_table/css/modal.css' %}">
```

Comment:
**Explanation.** Just trying to keep things simple.

<details>

We already know these files are specific to "admin_regis_table". The `type="text/css"` is not recommended.
> … given that CSS is the only stylesheet language used on the web, not only is it possible to omit the type attribute, but is actually now recommended practice. — [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-type)

</details>